docs/GettingStarted.md

```diff
@@ -1844,12 +1869,13 @@ The stats sent will include:
 | `runtime.ruby.thread_count` | `gauge` | Number of threads.                                       |
 | `runtime.ruby.gc.*`.        | `gauge` | Garbage collection statistics (one per value in GC.stat) |
 
-In addition, all metrics will include the following tags:
+Also, all metrics include the following tags:
```

Comment:
Nit pick but I think I prefer "in addition" to "also" (more formal.)

Reply:
I changed it back, but you should have a deep philosophical conversation with yourself about why on earth doc should be formal :D

Reply:
Haha, maybe I've just read too many specs.